README.md

```diff
@@ -143,7 +144,7 @@ grunt watch
 npm link
 
 cd Foo
-npm install --save git://github.com/turn/Smart-Table.git#v0.3.8
+npm install --save git://github.com/turn/Smart-Table.git#v0.3.12
```

Comment:
i see. I think the version in our readme should ignore patching number.
what do you think?